editor/src/utils/hashed-assets.ts

```diff
@@ -35,7 +35,7 @@ function getPossiblyHashedURLInner(url: string): string {
 
 export function getPossiblyHashedURL(url: string): string {
   const relativeURL = getPossiblyHashedURLInner(url)
-  return urljoin(STATIC_BASE_URL, relativeURL)
+  return appendToPath(STATIC_BASE_URL, relativeURL)
```

Comment:
Hmmm this is definitely less safe, so we'll have to make sure we're extra vigilant with uses of this function